ceph-releases/ALL/centos/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -4,7 +4,7 @@ bash -c ' \
   if [ -n "__GANESHA_PACKAGES__" ]; then \
     if [[ "${CEPH_VERSION}" == master || "${CEPH_VERSION}" == main ]]; then \
       ARCH=$(arch); if [[ "${ARCH}" == "aarch64" ]]; then ARCH="arm64"; fi ; \
-      curl -s -L "https://shaman.ceph.com/api/repos/nfs-ganesha/V5.7/latest/centos/__ENV_[DISTRO_VERSION]__/flavors/ceph_main/repo?arch=${ARCH}" -o /etc/yum.repos.d/ganesha.repo ; \
+      curl -s -L "https://shaman.ceph.com/api/repos/nfs-ganesha/V5-stable/latest/centos/__ENV_[DISTRO_VERSION]__/flavors/ceph_main/repo?arch=${ARCH}" -o /etc/yum.repos.d/ganesha.repo ; \
```

Comment:
Is this valid? https://shaman.ceph.com/api/repos/nfs-ganesha/V5-stable/ return `[]`.

Will this show up when we have stable builds?

Reply:
oh so we only build 5.5 and 5.7 https://shaman.ceph.com/builds/nfs-ganesha/. we need to start building V5-stable

Reply:
@ffilz did mention that v5-stable would be available.

Reply:
Once it actually is available on the ganesha side (it might already be, I haven't looked), you then have to update https://github.com/ceph/ceph-build/blob/main/nfs-ganesha/config/definitions/nfs-ganesha.yml#L41 to build 5-stable, wait for builds to start showing up in shaman, and then we can update ceph-container to point to it.